it/README.md

```diff
@@ -0,0 +1,81 @@
+# Licenze Creative Commons Licenses per progetti GitHub
+
+[![Main](https://img.shields.io/badge/main%20language-EN-blue)](/../../)
+
+In questo repo troverai dei modi facili di applicare le licenze Creative Commons ai repository attraverso il linguaggio Markdown.
+
+> **ATTENZIONE:**
+> Tu non dovrai usare le licenze Creative Commons per i programmi.
+> Usa le licenze Free Software per questi tipi di repository, come ad esempio GPL, BSD, ecc.
+> Le licenze Creative Commons sono pensate solo per le opere intellettuali.
+
+Il modo più semplice per trovare la giusta licenza CC per il tuo progetto è il
+[sito Creative Commons](https://creativecommons.org/choose/?lang=it).
+
+Qui vi presento due tipi di licenze CC:
+
+* [Creative Commons Attribution 4.0 International](#cc-attribution-40-international)
+
+* [Creative Commons Attribution-ShareAlike 4.0 International](#cc-attribution-sharealike-40-international)
+
+Ulteriori informazioni sulle licenze ed i testi delle licenze formattate in ```plain text``` possono essere trovate sul sito https://choosealicense.com/.
+
+Se tu vuoi scaricare una versione vettoriale delle immagini Creative Commons ed altre cose relative, visita il sito https://creativecommons.org/about/downloads/.
+
+
+## CC Attribution 4.0 International
+
+Per aggiungere una licenza CC BY al tuo progetto, ti basta aggiunge il seguente testo al tuo file `README.md`. Dovrai inoltre copiare il corrispondente [file di testo della licenza](LICENSE-CC-BY) e rinominarlo in `LICENSE`.
+
+```markdown
+Shield: [![CC BY 4.0][cc-by-shield]][cc-by]
+
+This work is licensed under a
+[Creative Commons Attribution 4.0 International License][cc-by].
```

Comment:
Could you translate these sentence also? Remember to change the name of the license